docs/admin/install/docker.rst

```diff
@@ -985,6 +994,10 @@ both Weblate and PostgreSQL containers.
 
     PostgreSQL password.
 
+.. envvar:: POSTGRES_PASSWORD_FILE
+
+    Path to file containing the PostgreSQL password. use as an alternative to POSTGRES_PASSWORD.
```

Comment:
Please do a small grammatic correction to:

*Path to the file containing the PostgreSQL password. Use as an alternative to POSTGRES_PASSWORD.*

Reply:
@opichon the article `the` is still missing. Can you copy-paste my suggestion?